src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
 
 const ChatEntry = (props) => {
+  const [likeButton, setLikeButton] = useState('🤍');
+  const toggleClicked = () => {
+    if (likeButton === '🤍') {
+      setLikeButton('❤️');
+      props.updateLikes(true);
+    } else {
+      setLikeButton('🤍');
+      props.updateLikes(false);
+    }
+  };
+
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">
+          <TimeStamp time={props.timeStamp} />
+        </p>
+        <button className="like" onClick={toggleClicked}>
```

Comment:
This is where we might be better served using the function we brought in from props.